test/common/runtime/runtime_features_test.cc

```diff
@@ -0,0 +1,53 @@
+#include <string>
+
+#include "envoy/api/v2/core/base.pb.h"
+
+#include "common/runtime/runtime_features.h"
+
+#include "test/mocks/runtime/mocks.h"
+#include "test/test_common/utility.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+using testing::NiceMock;
+using testing::Return;
+
+namespace Envoy {
+namespace Runtime {
+namespace {
+
+class FeatureFlagTest : public testing::Test {
+protected:
+  NiceMock<MockLoader> runtime_;
+};
+
+TEST_F(FeatureFlagTest, BasicTest) {
```

Comment:
Can you add a test for the default/empty proto to make sure the feature flag object does what it is supposed to do? (always return true).